src/sas/qtgui/Utilities/TabbedModelEditor.py

```diff
@@ -564,51 +685,168 @@ def getModel(self):
         """
         return self.tabWidget.currentWidget().getModel()
 
+    def addTab(self, filetype: str, name: str):
+        """
+        Add a tab to the tab widget
+        :param filetype: filetype of tab to add: "python" or "c"
+        :param name: name to display on tab
+        """
+        if filetype in ["python", "py", ".py"]:
+            self.editor_widget = ModelEditor(self, is_python=True)
+            self.tabWidget.addTab(self.editor_widget, name)
+            self.editor_widget.modelModified.connect(self.editorModelModified)
+        elif filetype in ["c", ".c"]:
+            self.c_editor_widget = ModelEditor(self, is_python=False)
+            self.tabWidget.addTab(self.c_editor_widget, name)
+            self.c_editor_widget.modelModified.connect(self.editorModelModified)
+
+    def removeTab(self, filetype: str):
+        """
+        Remove a tab from the tab widget.
```

Comment:
The two cases where `removeTab` is called (lines 185 and 92) use integers as arguments (`1` and `0`).
How is this going to work with this signature?

Reply:
Those instances are removing tabs from tabWidget, not calling this method. This isnt called anywhere, so I will likely remove it.